triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -440,14 +441,12 @@ abstract class AbstractFuncTests
             QualifiedName.assertFromString("TemplateIdFilter:testOne"),
             party,
           )
+          _ <- create(client, party, one(party))
+          _ <- create(client, party, two(party))
           (acs, offset) <- runner.queryACS()
-          // 2 for the creates from the test
-          // 2 for the completions from the test
           // 1 for the create in the trigger
           // 1 for the completion from the trigger
-          finalStateF = runner.runWithACS(acs, offset, msgFlow = Flow[TriggerMsg].take(6))._2
-          _ <- create(client, party, one(party))
-          _ <- create(client, party, two(party))
+          finalStateF = runner.runWithACS(acs, offset, msgFlow = Flow[TriggerMsg].take(2))._2
           _ <- finalStateF
```

Comment:
Why do you prefer not inlining `finalStateF` ?

Reply:
I don’t, just missed the fact that after reshuffling they’re right next to each other. changed it